keyboards/sixshooter/sixshooter.h

```diff
@@ -10,17 +10,6 @@
 	{ K00, K01, K02, K03, K04, K05 }, \
 }
 
-
-/*
- * Define keyboard specific keycodes for controlling on/off for all LEDs as they
- * are all on different pins with this PCB, rather than a single backlight pin
- */
-enum keyboard_keycode {
```

Comment:
Honestly, this is alright, but you'd want to note in the default keymap that `SAFE_RANGE_KB` is the range that you need to use.